paf-mvp-figma/audit/README.md

```diff
@@ -0,0 +1,53 @@
+![OneKey Logo](images/onekey-logo.svg)
+
+# Audit Log UI design
+
+Hello! :wave:
+
+You are reading about a user interface design for a modern advertising funded
+open web.
+
+The audit log shows users the organizations that used data to deliver the
+content and services they interact with. The UI is both understandable and easy
+to use, whilst providing detailed information for those users that want it. The
+UI also supports providing audit log level information to agents that can
+analyse the data on behalf of users.
+
+The Figma file is available [here](onekey-audit-log-demo.fig).
+
+# Features
+
+## Advert audit
+
+On the first screen, the user receives a summary of the audit results. There are
+three possible results:
+
+-   Everything is OK
+-   There is an irregularity (error, missing data, or other) in the transactions
+    responsible for the ad display.
+-   The user's preferences may not have been respected.
+
+## User data
+
+They can also see their marketing preference and if they have generated a
+synchronization between devices in the CMP settings, the code for this will be
+shown.
+
+## Participating companies
+
+The user can see which companies used data involved in the display of the ad
+they saw. The companies are listed in different tabs to aid the user in
+identifying the role they played in serving the advert. The company's terms for data use, and contact details are shown enabling the user to directly 
+communicate with these companies should they need to.
+
+## Download
+
+Users can download the content of the audit, which they can use to contact the
+authorities if they believe their rights have been violated.
+
+![OneKey Download](images/download.png)
+
+## About the authors
+
+[Dot Creative](https://dotcreative.hu) is a Digital Agency. Our main profile is
```

Comment:
I'm confused about this whole directory: what is its purpose? why a documentation directory in this implementation repo? I'm not sure why there is such a need for such README, but if there is, I think it should be in (https://github.com/prebid/addressability-framework)

Reply:
The `.fig` file is the one that contains all the design work which is the input into the pattern library. For those working on the design this file is important to be able to apply updates and understand the work. It is not relevant to developers working on code, but is relevant for the overall project. It could be moved to addressability-framework if that is the better repository. Please let me know once you've considered the `.fig` file.

Reply:
Yes, I think this would be better placed in the other repository, because it is focused on specs and design, while this one is focused on implementation.
WDYT?